OutOfSchool/OutOfSchool.ElasticsearchData/OutOfSchool.ElasticsearchData.csproj

```diff
@@ -28,6 +28,7 @@
 
   <ItemGroup>
     <ProjectReference Include="..\OutOfSchool.Common\OutOfSchool.Common.csproj" />
+    <ProjectReference Include="..\OutOfSchool.DataAccess\OutOfSchool.DataAccess.csproj" />
```

Comment:
Better move ownershiptype to common project or create similar enum (and map it) in ES project. Don't need to pull entire sql dataaccess layer to ES